cmd/s2i/main.go

```diff
@@ -212,7 +212,8 @@ func newCmdRebuild(cfg *api.Config) *cobra.Command {
 				os.Exit(1)
 			}
 
-			if r, err := os.Open(cfg.DockerCfgPath); err == nil {
+			r, err := os.Open(cfg.DockerCfgPath)
+			if err == nil {
```

Comment:
There were few remaining goshadow errors so I decided to tackle them.  This wasn't a case in question but I changed it to match the other changes in the same function stylistically.